src/mol-gl/shader/chunks/common-frag-params.glsl.ts

```diff
@@ -157,7 +157,7 @@ float fbm(in vec3 p) {
         #elif defined(dXrayShaded_inverted)
             alpha *= pow(abs(dot(normal, vec3(0.0, 0.0, 1.0))), uXrayEdgeFalloff);
         #endif
-        return alpha;
+        return min(max(alpha, 0.001), 0.999);
```

Comment:
`clamp` may be used instead
```suggestion
        return clamp(alpha, 0.001, 0.999);
```

Reply:
Thanks @papillot, changed